core/primitives/rle_bitset/rle_bitset.hpp

```diff
@@ -56,6 +75,24 @@ namespace fc::primitives {
       }
       return result;
     }
+
+    inline bool contains(const RleBitset &other) {
```

Comment:
both this and other are sorted.
if needed, this method can be optimized to single pass loop without find.